about.html

```diff
@@ -8,11 +8,36 @@
     <title>E-learning Online Educational Courses</title>
   </head>
   <body>
-    <header>      
+    <header>
+      <div class="header-social">
+        <ul>
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My page</a></li>
+          <li><a href="#">Log out</a></li>
+        </ul>
+      </div>
+      <nav class="mobile-nav">
+        <img class="hamburger" src="./resources/images.png" alt="menu-icon" />
+        <a href="#"
+          ><img class="logo" src="./resources/logo22.jfif" alt="logo-image"
+        /></a>
+        <ul class="desktop-ul">
+          <li><a class="red-link" href="./about.html">About</a></li>
+          <li><a class="grey-link" href="#attract">Attractions</a></li>
+          <li><a class="grey-link" href="#events">Events</a></li>
+          <li>
+            <a class="grey-link" href="#partner">Partner</a>
+          </li>
+          <li><a class="grey-link" href="#contact">Contact</a></li>
+          <li><a class="red-link" href="#news">OYE Campaign</a></li>
```

Comment:
- [ **OPTIONAL** ] Good job making all the changes required by the previous reviewer, your project looks perfect. But. what do you think if you add a link to your nav-links of the `about.html` that can enable users to access the **home** page from the `About` page( a thing like this: `<li><a class="grey-link" href="./index.html">Home</a></li>`).

Reply:
Yes, thanks. User can access the hompage through the logo.